straxen/plugins/peaklets/peaklet_classification.py

```diff
@@ -12,7 +12,7 @@
 class PeakletClassification(strax.Plugin):
     """Classify peaklets as unknown, S1, or S2."""
 
-    __version__ = "3.0.3"
+    __version__ = "3.0.4"
```

Comment:
I have no problems with the change here, but out of curiosity, why did we return peaklet_classification as a dict before? Also, @FaroutYLq , will this be an issue for reprocessing if the version changes but the output is functionally the same?

Reply:
Thanks for the headup @JYangQi00 , very important catch! I would try to avoid touching the standard `peaklet_classification` in this SOM PR. It doesn't look to me that the performance of `peaklet_classification` is meaningfully changed, and if it is just about style we should also have a different PR for this. I would try to avoid bumping version which changes lineage so we have to reprocess again the standard pipeline.

Reply:
The change is coming from #1299 and indeed the version bump is purely optional. Since I do not modify the outcome of the plugin we can also merge without version bump if you guys prefer. If you agree I change #1299.